ui/l10n/en.js

```diff
@@ -1484,6 +1485,9 @@ var dictionary = {
 "label.rule.number":"Rule Number",
 "label.rule.number.short": "#Rule",
 "label.rules":"Rules",
+"label.run.diagnostics.type":"Type",
+"label.run.diagnostics.destination":"Destination",
+"label.run.diagnostics.extra":"Extra Args",
```

Comment:
Spelling out the entire word looks more professional. Maybe just call it 'Arguments'?

Reply:
Changed Extra Args -> Extra Arguments